Source/Views/TableViews/SPCopyTable.m

```diff
@@ -1234,13 +1234,19 @@ - (BOOL)shouldUseFieldEditorForRow:(NSUInteger)rowIndex column:(NSUInteger)colIn
 {
 	// Return YES if the multiple line editing button is enabled - triggers sheet editing on all cells.
 	if ([prefs boolForKey:SPEditInSheetEnabled]) return YES;
-	
+
+    NSUInteger editInSheetForLongTextLengthThreshold = NSUIntegerMax;
```

Comment:
I would have like 15 here (the default), and add the value from prefs to the if condition too and assign it only if it exists - current implementation could lead to getting undefined value from prefs that could cause a crash

Reply:
done